CMSgov/mint-app
# [EASI-3771] - Quality performance impacts payment

# EASI-3771

## Changes and Description

**BE Changes**
- Added new enum type `QualityPerformanceImpactType`
- Added migration to map `Bool` to `QualityPerformanceImpactType`
- Added new field `qualityPerformanceImpactsPaymentOther` when `QualityPerformanceImpactType.OTHER` is selected
- Added to Postman collection
- Added server test

**FE Changes**
- Added/updated translation for `qualityPerformanceImpactsPayment` and `qualityPerformanceImpactsPaymentOther`
- Updated form component for `opsEvalAndLearning`
- Updated readonly
- Updated unit tests



## How to test this change

- Navigate to page 7 of 9 for `opsEvalAndLearning` (Need to make sure the conditional question triggers the `Quality questions` section of this page)
- Verifiy data persistence
- Verifiy readonly

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
